replay/tsconfig.json

```diff
@@ -1,9 +1,10 @@
 {
   "compilerOptions": {
-    "outDir": "dist",
+    "outDir": "../build/replay",
     "module": "esnext",
     "target": "es2018",
     "moduleResolution": "node",
+    "composite": true,
```

Comment:
Why is this needed?

Reply:
I read up on `composite` in the TS documentation, but I"m not quite sure why we need it here.

Reply:
Using the project references requires it